core/src/main/java/com/datastrato/gravitino/schema/SupportsSchemas.java

```diff
@@ -18,10 +18,12 @@
 // Referred from Apache Spark's connector/catalog implementation
 // sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/SupportNamespaces.java
 
-package com.datastrato.gravitino.rel;
+package com.datastrato.gravitino.schema;
```

Comment:
Can you please move this to `connector` module, this `SupportsSchemas` will be used by catalog implementor, so we need to move this to that module.

Reply:
Done